packages/connect-web/src/browserstacktests/README.md

```diff
@@ -0,0 +1,11 @@
+# Connect-Web Browserstack Tests
```

Comment:
Note these are at `src/browserstacktests` and conformance is at the project root, which feels odd. I kept them here to mimic how it was in `connect-web-test`.

Reply:
Agree it would be nice. I think it should be possible to move the browserstack directory up by modifying `karma.browserstack.conf.cjs`. I think it's better to try that separately.